src/main/java/core/basesyntax/Track.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Track extends Machine {
+
+    @Override
+    public void doWork() {
```

Comment:
```suggestion
public class Track extends Machine {
    @Override
    public void doWork() {
```
same in other classes